packages/store/addon/-private/system/store/record-data-store-wrapper.ts

```diff
@@ -53,8 +54,11 @@ export default class RecordDataStoreWrapper implements IRecordDataStoreWrapper {
   }
 
   notifyErrorsChange(modelName: string, id: string | null, clientId: string | null) {
-    let internalModel = this._store._getInternalModelForId(modelName, id, clientId);
-    internalModel.notifyErrorsChange();
+    let internalModel = internalModelFactoryFor(this._store).peekId(modelName, id, clientId);
+
+    if (internalModel) {
```

Comment:
We should probably assert if no IM here, that doesn't seem like a valid code path